Client/Frontend/Browser/BackForwardListView.swift

```diff
@@ -0,0 +1,196 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+import WebKit
+import Storage
+import SnapKit
+
+class BackForwardListView: UIView, UITableViewDataSource, UITableViewDelegate {
```

Comment:
I'm not sure subclassing `UIView` here is the best approach for this feature. Generally we want to keep our UIView's as lean as possible by keeping any 'controller' like functionality out of it. This feels more like a `UIViewController` than a `UIView` in it's current state. 

Also, extending the table view protocols on the UIView adds more controller logic to this view that we can probably factor out into new classes. Let's jam on this approach later.